### PR TITLE
ADEN-11063 Fix transition from embedded big to embedded resolved

### PR DESCRIPTION
**Pull request by preludek**

### CODE REVIEW

**Review by Tomasz-at-Fandom — APPROVED** [review]

File: platforms/shared/templates/handlers/slot/slot-decision-embedded-big-to-embedded-resolved-handler.ts [hunk 1]
```diff
@@ -1,20 +1,33 @@
...
+		this.playerRegistry.video$
+			.pipe(
+				switchMap(({ player }) => {
+					this.isVideo = true;
+					return fromEvent(player, 'wikiaAdCompleted');
+				}),
+				tap(() => transition('embeddedResolved')),
+				takeUntil(this.unsubscribe$),
+			)
+			.subscribe();
```
Comment (Tomasz-at-Fandom) [inline platforms/shared/templates/handlers/slot/slot-decision-embedded-big-to-embedded-resolved-handler.ts:25]: Please put this `subscription` in to a separate private method. Maybe `videoTransition()` ?

File: platforms/shared/templates/handlers/slot/slot-decision-embedded-big-to-embedded-resolved-handler.ts [hunk 2]
```diff
@@ -1,20 +1,33 @@
...
+
 		this.timeout
```
Comment (Tomasz-at-Fandom) [inline platforms/shared/templates/handlers/slot/slot-decision-embedded-big-to-embedded-resolved-handler.ts:27]: Please put this subscription in to a separate private method. Maybe staticTransition() ?

**Review by Tomasz-at-Fandom — CHANGES_REQUESTED** [review]

File: platforms/shared/templates/handlers/slot/slot-decision-embedded-big-to-embedded-resolved-handler.ts (hunk 2 above)
Comment (Tomasz-at-Fandom) [reply platforms/shared/templates/handlers/slot/slot-decision-embedded-big-to-embedded-resolved-handler.ts:27]: I could not mark the whole block of code, for some reason, this is for the lines 27-34 on the feature branch.

File: platforms/shared/templates/handlers/slot/slot-decision-embedded-big-to-embedded-resolved-handler.ts (hunk 1 above)
Comment (preludek) [reply platforms/shared/templates/handlers/slot/slot-decision-embedded-big-to-embedded-resolved-handler.ts:25]: Good idea. Done.

File: platforms/shared/templates/handlers/slot/slot-decision-embedded-big-to-embedded-resolved-handler.ts (hunk 2 above)
Comment (preludek) [reply platforms/shared/templates/handlers/slot/slot-decision-embedded-big-to-embedded-resolved-handler.ts:27]: Good idea. Done.

**Review by Tomasz-at-Fandom — APPROVED** [review]

**Pull request merged**